make_pao1_pa14_compendia/0_decide_threshold.py

```diff
@@ -0,0 +1,215 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.9.1
+#   kernelspec:
+#     display_name: Python [conda env:core_acc]
+#     language: python
+#     name: conda-env-core_acc-py
+# ---
+
+# # Decide threshold
+#
+# The goal of this notebook is to define a threshold to determine if a sample if PAO1 or not (likewise, if a sample is PA14 or not). We used known labels from SRA to do this. Specifically, we examined the distribution of PAO1 samples (grey) vs non-PAO1 samples (blue). We define the threshold to be one that separated between the two distributions. We use this threshold in [1_create_compendia.ipynb](1_create_compendia.ipynb) to partition gene expression data into PAO1 and PA14 compendia.
+
+# %load_ext autoreload
+# %autoreload 2
+# %matplotlib inline
+import os
+import pandas as pd
+import seaborn as sns
+from textwrap import fill
+import matplotlib.pyplot as plt
+import utils
+
+# +
+# Raw (TPM) expression data files
+pao1_expression_filename = "../qc_filtering/qc-out/pao1_aligned_compendium_p2_filtered_counts_norm.csv" 
+pa14_expression_filename = "../qc_filtering/qc-out/pa14_aligned_compendium_p2_filtered_counts_norm.csv"
+
+# File containing table to map sample id to strain name
+sample_to_strain_filename = "Run_Table_Strain_Bool_GD.csv"
```

Comment:
I should submit a PR with the code I used to make this so you can see it, it's short and sweet but will be good to have.